staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/grpc_service_unix_test.go

```diff
@@ -138,7 +138,7 @@ func TestTimeouts(t *testing.T) {
 
 				service, err = NewGRPCService(socketName.endpoint, tt.callTimeout)
 				if err != nil {
-					t.Fatalf("failed to create envelope service, error: %v", err)
+					t.Errorf("failed to create envelope service, error: %v", err)
```

Comment:
this test seems like an ideal place to nest, defer the `wg.Done`, pipe errors to a channel, check the channel after the wait.

Reply:
> this test seems like an ideal place to nest, defer the `wg.Done`, pipe errors to a channel, check the channel after the wait.

I edit it just fix "SA2002 | Called testing.T.FailNow or SkipNow in a goroutine, which isn't allowed"

Reply:
> this test seems like an ideal place to nest, defer the `wg.Done`, pipe errors to a channel, check the channel after the wait.

 I get it